src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ exporter.
   [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.23.0/specification/metrics/sdk_exporters/otlp.md#additional-configuration).
   ([#4667](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4667))
 
+* **[Experimental feature](./README.md#experimental-features)**
```

Comment:
@alanwest The main README lists multiple ways to do experimental features and then this links to that. Users won't know which type of experimental feature style this happens to use (pre-release build in this case). Could we write the main README with sections on the different types so we could link to an anchor from here? Or some other way you can come up with to make it clear 😄

Reply:
+1
It would be good to first clarify how we are categorizing experimental features here:

1) Experimental features offered via public APIs.
2) Experimental features offered via feature flag that change the internal code behavior.

Also, to me this is not a experimental feature just yet as I do not have a way to opt-in/out :)

Reply:
> The main README lists multiple ways to do experimental features and then this links to that.

To clarify, this does not link to the main readme. Did you see the OTLP exporter readme? It documents which style this feature is provided by.

> Also, to me this is not a experimental feature just yet as I do not have a way to opt-in/out :)

It's experimental in that it's not gonna be in the next stable release. I've attempted to document things as they will be when this PR merges to main. If in a future PR, someone introduces a feature flag or something for this, then the documentation will need to be updated.

I'm open to making things more clear since that's my main goal with this documentation, but please make a github suggestion proposing different language or formatting, so I can be clear what you find unclear 😄.

Reply:
> To clarify, this does not link to the main readme. Did you see the OTLP exporter readme? It documents which style this feature is provided by.

Ah sorry my bad! Looks good.

Reply:
Something like https://github.com/open-telemetry/opentelemetry-dotnet/pull/4762/files#r1291758235?